sdk/identity/ci.yml

```diff
@@ -30,11 +30,7 @@ extends:
         safeName: azureidentity
       - name: azure-identity-cache-persistence
         safeName: azureidentitycachepersistence
-        # TODO: Remove when REX validation tool is integrated: https://github.com/Azure/azure-sdk-for-js/issues/26770
-        skipPublishDocMs: true
       - name: azure-identity-broker
         safeName: azureidentitybroker
       - name: azure-identity-vscode
         safeName: azureidentityvscode
-        # TODO: Remove when REX validation tool is integrated: https://github.com/Azure/azure-sdk-for-js/issues/26770
-        skipPublishDocMs: true
```

Comment:
this package is I believe still broken with no immediate remediation plans, so I think we should still leave it unpublished though we can remove the comment. /cc @KarishmaGhiya